libs/containers/Interaction.lua

```diff
@@ -65,9 +65,12 @@ function Interaction:__init(data, parent)
     -- Last resort, request the channel object from the API if wasn't cached
     if not self._channel then
       local channel = self._api:getChannel(channelId)
-      if not channel then goto skip end -- somehow channel not available
+      if not channel then goto skip end
 
       local guild = channel.guild and parent._guilds:_insert(channel.guild)
+      if (not guild) and (channel.guild_id) then
+        guild = parent:getGuild(channel.guild_id)
+      end
       if guild then
         self._channel = guild._text_channels:_insert(channel)
```

Comment:
This here absolutely becomes invalid, if a guild is available (and it was explicitly made available in case of VoiceTextChannel), how come it is getting inserted into `_text_channels` cache?